Misc/NEWS.d/next/Core and Builtins/2022-09-27-03-52-49.gh-issue-86529.rWk1tr.rst

```diff
@@ -0,0 +1 @@
+enhance _check_running() ValuError error message
```

Comment:
Add state the ```_check_running()```  error message. 
```suggestion
Add state to the error message of the :meth:`multiprocessing.Pool._check_running` method in the :mod:`multiprocessing` module.

Reply:
The NEWS entry doesn't mention the module name (multiprocessing) and should not reference a private function (users don't see it). Honestly, I think that you should just remove the NEWS entry. Enhancing an error message is nice, but doesn't require to be documented in the Changelog.